src/tasks/task_test_1.h

```diff
@@ -0,0 +1,18 @@
+#ifndef TASKS_TASK_TEST_1_H
+#define TASKS_TASK_TEST_1_H
+
+#include "core/task_mngr.h"
+#include "utils/logger.h"
+
+
+
+void task_test_1_run(task_arguments_t* argv);
+void task_test_1_init(task_arguments_t* argv);
+void task_test_1_finish(task_arguments_t* argv);
+void task_test_1_resume(task_arguments_t* argv);
+void task_test_1_calculate_priority(task_arguments_t* argv);
```

Comment:
These functions produce warning about `argv` being an unused parameter. Should be fixed by implementing an explicit makefile rule just for the `task_test_1.c` and `task_test_2.c`.